averages.js

```diff
@@ -0,0 +1,38 @@
+var pry = require('pryjs');
+
+Array.prototype.total = function () {
+  var total = 0;
+
+  for(i = 0; i < this.length; i++ ) {
+    var total = total + this[i]
+  }
+  return total;
+};
+
+Array.prototype.mean = function () {
+  return (this.total() / this.length);
+};
+
+Array.prototype.median = function () {
+  var sorted = this.sort(function (a, b) {
+    return a - b
+  });
+  var length = this.length;
+
+  if(length % 2 === 0) {
+    return (sorted[length/2] + (sorted[length/2 - 1])) / 2
+  } else {
+    return sorted[Math.floor(length/2)]
+  }
+};
+
+Array.prototype.mode = function() {
+  var obj = {}
```

Comment:
Similar to Ruby-land, let's try to name variables without using their data types. This could be something like `numberCounts` or `numberFrequencies`